members/S001194.yaml

```diff
@@ -189,15 +178,12 @@ contact_form:
             - Welfare
             - "Women's Issues"
     - click_on:
-        - selector: input#response_y
-    - wait:
-        - value: 1
+        - selector: "label[for='input-3048820B-A361-6166-70F4-2D74C1E208F4']"
     - click_on:
-        - value: Send
-          selector: input#side-search-btn
+        - selector: "div > div > button[type='submit']"
```

Comment:
is `button[type='submit']` alone not enough? this positional selector is pretty non-specific and also weak to any sort of structural change

Reply:
yeah, that works 👍🏻